src/parsers/manifest/dash/parse_representations.ts

```diff
@@ -29,8 +29,49 @@ import {
   IRepresentationIntermediateRepresentation,
 } from "./node_parsers/Representation";
 import { IParsedSegmentTemplate } from "./node_parsers/SegmentTemplate";
+import { IScheme } from "./node_parsers/utils";
 import parseRepresentationIndex from "./parse_representation_index";
 
+/**
+ * Combine unique inband event streams from representation and
+ * adaptation data.
+ * @param {Object} representation
+ * @param {Object} adaptation
+ * @returns {undefined | Array.<Object>}
+ */
+function getAcceptedInbandEventStreams(
```

Comment:
That whole function looks too complicated for nothing:
  1. Is it that problematic to have duplicated members in here?
  2. does that even happen?
  3. Even so, I feel like this is too complicated for nothing.

Reply:
Also the name of the function doesn't really tell what it's doing.

Reply:
Yes indeed, there should not be issues with the fact of duplicating InbandEventStreams.